Jahsie/template/css/main.css

```diff
@@ -0,0 +1,138 @@
+/* CSS for GUI...*/
+@import url(http://fonts.googleapis.com/css?family=Lato:400,900);
+html {
+  font-size: 12px; }
+
+@media screen and (min-width: 900px) {
+  body {
+    font-size: 16px; } }
+@media screen and (min-width: 1200px) {
+  html {
+    font-size: 18px; } }
+body {
+  font-family: 'Lato', sans-serif;
+  color: grey; }
+  body p {
+    text-align: center;
+    font-size: .85rem; }
+
+#main {
+  /*	width: 90%;
+  	height: 90%;
+  	overflow: auto;
+  	margin: auto;
+  	position: absolute;
+  	top: 0; left: 0; bottom: 0; right: 0;
+  */
+  /*top: 50%;
+  -webkit-transform: translateY(-50%);
+  -ms-transform: translateY(-50%);
+  transform: translateY(-50%);*/ }
+
+h1 {
+  font-weight: 900;
+  text-align: center; }
+
+#intro {
+  position: relative;
+  box-sizing: border-box;
+  display: inline-block;
+  width: 20%;
+  float: left;
+  text-align: center;
+  padding: 3% 0px 0px 3%;
+  height: 100%; }
+
+.guts {
+  position: absolute;
+  bottom: 2px;
+  display: block;
+  margin: 0px auto;
+  width: 80%; }
+
+.quote {
+  font-style: italic;
+  vertical-align: bottom; }
+
+#signature {
+  margin: 50px 0px; }
+
+#memorygame {
+  height: 100%;
+  position: relative;
+  box-sizing: border-box;
+  display: inline-block;
+  float: right;
+  width: 80%;
+  margin: auto;
+  padding: 3% 3% 0px 3%; }
+
+button {
+  font-size: 15px;
+  background-color: transparent;
+  height: 50px;
+  width: 80px;
+  color: grey;
+  border: 3px solid grey; }
+
+button:hover {
+  background-color: grey;
+  color: white; }
+
+.cardDiv:hover {
+  opacity: 50%;
+  outline: 1px solid white;
+  outline-offset: -10px; }
+
+img {
+  max-width: 100%; }
+
+#cardSet {
+  width: 100%;
+  /*	position: absolute;*/
+  top: 50%; }
+
+.cardDiv {
+  background-size: contain;
+  /*background-repeat: none;*/
```

Comment:
You can get the background images to stop repeating with `background-repeat: no-repeat`. This will leave a white space if the div containing the image is larger than the image. You could deal with that in a couple of different ways. The easiest would be to just lock the dimensions of the divs to the size of the images. Alternately, you could give the divs a `background-color` that matches the background color of the image. Yet another option would be doing the card backs as text. You might even be able to use `calc` in your CSS to dynamically resize the font based on the width of the containing div, but that's not something I've tinkered with.